src/css/jwplayer/flags/fullscreen.less

```diff
@@ -1,14 +1,17 @@
 .jwplayer.jw-flag-fullscreen {
-    /* !important required to ensure that fullscreen state is respected */
```

Comment:
Hmm, I'm wondering whether the width and height should be duplicated here, rather then moved? /cc @radium-v

Reply:
Yeah. Which element is fullscreened now? If it's still the player container couldn't it clip the wrapper? (overflow hidden)